packages/forms/src/UIForm/utils/widgets.js

```diff
@@ -47,13 +47,18 @@ const widgets = {
 	fieldset_text: FieldsetTextMode,
 
 	// fields: text mode
+	button_text: () => null,
```

Comment:
maybe we can name it VoidComponent or smtg like that, and add the relevant UT on it

Reply:
Add storybook with all field in text mode.
Add support for all existing fields